dm/dumpling/dumpling.go

```diff
@@ -125,6 +125,17 @@ func (m *Dumpling) Process(ctx context.Context, pr chan pb.ProcessResult) {
 		m.mu.Lock()
 		m.core = dumpling
 		m.mu.Unlock()
+
+		failpoint.Inject("dumpUnitProcessForever", func() {
```

Comment:
we already have this failpoint in [here](https://github.com/pingcap/tiflow/blob/101d53fe2c5cbabf6ec1851319f1c7223bbb4239/dm/dumpling/dumpling.go#L93)